Production/python/scenarios.py

```diff
@@ -60,6 +60,13 @@ def __init__(self,sname):
                           jecfile="data/jec/Summer16_23Sep2016AllV3_DATA",
                           residual=True,
                           era="Run2_25ns")
+        elif sname == "2017PromptCollisions":
+            self.set_vars(globaltag="92X_dataRun2_Prompt_v8",
+                          tagname="RECO",
+                          jsonfile="data/Cert_294927-299042_13TeV_PromptReco_Collisions17_JSON.txt",
+                          jecfile="data/jec/Summer16_23Sep2016AllV3_DATA",

```

Comment:
I doubt this is the right JEC for 2017 data... it actually doesn't look like there are 2017 JECs at this point, so probably best to leave this as `""` for now (which will avoid reapplying JECs at all, and just use the online)